ops/cmd/print_staging_report/main.go

```diff
@@ -93,82 +93,85 @@ func PrintStagingReport(cliCtx *cli.Context) error {
 		return fmt.Errorf("root directory error: %w", err)
 	}
 
-	chainCfg, err := manage.StagedChainConfig(wd)
-	if errors.Is(err, manage.ErrNoStagedConfig) {
-		output.WriteOK("no staged chain config found, exiting")
-		return nil
-	}
-	if err != nil {
-		return fmt.Errorf("failed to get staged chain config: %w", err)
-	}
+	stagedChainCfgs, err := manage.StagedChainConfigs(wd)
```

Comment:
We could test this command out by running 

```
just import-devnet ../devnets/alphanets/interop-rc-alpha/op-deployer/state.json ../devnets/alphanets/interop-rc-alpha/manifest.yaml
```
and raising a PR.

Reply:
This feels like a bit of distinct scope that we can pinch of to allow work to get unblocked and keep changesets small and easy to review. Filed an issue here #1017